tests/warp/warp_test.go

```diff
@@ -318,32 +340,9 @@ var _ = ginkgo.Describe("[Warp]", ginkgo.Ordered, func() {
 		gomega.Expect(err).Should(gomega.BeNil())
 		defer sub.Unsubscribe()
 
-		// Trigger building of a new block at the current timestamp.
-		// This timestamp should be after the ProposerVM activation time or ApricotPhase4 block timestamp.
-		// This should generate a PostForkBlock because its parent block (genesis) has a timestamp (0) that is greater than or equal
-		// to the fork activation time of 0.
-		// Therefore, when we build a subsequent block it should be built with BuildBlockWithContext
 		nonce, err := chainBWSClient.NonceAt(ctx, fundedAddress, nil)
 		gomega.Expect(err).Should(gomega.BeNil())
 
```

Comment:
can we clean  lines 343 and 339 up since they are no longer used?

Reply:
The nonce is used in the predicate tx below.